### ISSUE TITLE
Fix deprecation warnings

### ISSUE THREAD

**Issue by bmuschko** [jvoegele/gradle-android-plugin#49]
I was running my project in the emulator with Gradle 1.0-milestone-8 and got this deprecation warning:

```
fileTree(Closure) is a deprecated method. Use fileTree((Object){ baseDir }) to have the closure used as the file tree base directory
```

I think it would be a good idea to fix it to harden it for future Gradle versions.

**Comment by ealden** [comment]
Thanks for the bug report.  I've upgraded to 1.0-milestone-8 but haven't tried running Gradle Android projects with it.  Will see if we can get some PR to fix all the warnings with it.

**Comment by fdasoghe** [comment]
You're right, of course. The project is a bit... slowed recently, anyway
I'll try to fix this (and other issues) if I'll find the time. Of course,
any pull request would help a lot ;o)

Cheers,

Fabio

2012/2/20 Benjamin Muschko <
reply@reply.github.com

**Comment by bmuschko** [comment]
Here you go: https://github.com/jvoegele/gradle-android-plugin/pull/50

**Comment by ealden** [comment]
Merged #50 to master.  Thanks for the PR @bmuschko !
